commands/list_models.go

```diff
@@ -118,6 +118,62 @@ func innerAppConfigValidateCommand(cnf *config.Config) Command {
 	}
 }
 
+type Region struct {
+	Label        string `json:"label"`
+	Zone string `json:"zone"`
+}
```

Comment:
The main problem with this is that it would depend on #82 or similar (because the `/regions` API requires authentication)